SingularityUI/app/views/tail.coffee

```diff
@@ -40,52 +40,68 @@ class TailView extends View
             @$el.html "<h1>Could not get request file.</h1>"
 
     render: =>
-        breadcrumbs = utils.pathToBreadcrumbs @path
+        @collection.fetchMagicString().done (res) =>
+            isImage = /PNG|ÿØÿà|GIF89a|GIF87a/.test(res.data)
```

Comment:
@simplyianm That's a JPEG's signature, http://en.wikipedia.org/wiki/Magic_number_%28programming%29